test/test_joint_trajectory_splitter.py

```diff
@@ -0,0 +1,305 @@
+#! /usr/bin/env python
+
+import rospy
+import actionlib
+import control_msgs.msg
+import roslaunch
+import rospkg
+import trajectory_msgs.msg
+import pytest
+import copy
+from giskardpy import logging
+
+@pytest.fixture(scope=u'function')
+def init_ros():
+    rospy.init_node('JointTrajectorySplitterTest')
+
+
+@pytest.fixture(scope=u'function')
+def giskard_package_path():
+    rospack = rospkg.RosPack()
+    giskard_package_path = rospack.get_path('giskardpy')
+    return giskard_package_path
+
+@pytest.fixture(scope=u'function')
+def simple_trajectory_goal():
```

Comment:
You don't need fixtures for functions that don't setup anything. This might as well be a normal function. Same for the other goal functions.